lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm

```diff
@@ -0,0 +1,109 @@
+#SpeedseqSv.pm
+
+
+package Genome::Model::Tools::DetectVariants2::SpeedseqSv;
+use warnings;
+use strict;
+
+use Genome;
+use File::Basename qw(basename dirname);
+use File::Spec;
+use IPC::System::Simple;
+use Genome::File::Tsv;
+
+class Genome::Model::Tools::DetectVariants2::SpeedseqSv {
+    is => 'Genome::Model::Tools::DetectVariants2::Detector',
+};
+
+	my $temp_directory = Genome::Sys->create_temp_directory();
+	my $pkg = 'Genome::Model::Tools::Speedseq::Sv';
```

Comment:
This might benefit from being a constant classwide property, like:
```perl
has => [
  gmt_package => {
    is => 'Text',
    is_classwide => 1,
    is_constant => 1,
    value => 'Genome::Model::Tools::Speedseq::Sv',
  },
],
```
(There are likely better name choices than `gmt_package` :smile: )